docs/core/advanced_tutorials/using-result-handlers.md

```diff
@@ -0,0 +1,133 @@
+---
```

Comment:
I have mostly small suggested changes for this file, feel free to incorporate as many or few of them as you wish: https://github.com/PrefectHQ/prefect/compare/issue-1792-add-code-ex-to-ResultHander...issue-1792-add-code-ex-to-ResultHander-alex-comments?expand=1

Reply:
Thanks, I took them all! The line targeting highlighting is nice!